From c9a4de34c1d55e3f2b3d7cdd0736673c0a575f46 Mon Sep 17 00:00:00 2001
From: Anson Huang <Anson.Huang@nxp.com>
Date: Thu, 21 Jun 2018 08:59:14 +0800
Subject: [PATCH 4055/5242] MLK-18648 thermal: improve imx sc thermal driver
 name

commit  5416af1981172c45bbac0b13e2db682ae32380b6 from
https://source.codeaurora.org/external/imx/linux-imx.git

Improve i.MX system controller thermal driver name
by making it lower case.

Signed-off-by: Anson Huang <Anson.Huang@nxp.com>
Reviewed-by: Bai Ping <ping.bai@nxp.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 drivers/thermal/imx_sc_thermal.c |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/thermal/imx_sc_thermal.c b/drivers/thermal/imx_sc_thermal.c
index 931fa7c..21f8c2a 100644
--- a/drivers/thermal/imx_sc_thermal.c
+++ b/drivers/thermal/imx_sc_thermal.c
@@ -221,7 +221,7 @@ static int imx_sc_tsens_remove(struct platform_device *pdev)
 		.probe = imx_sc_tsens_probe,
 		.remove = imx_sc_tsens_remove,
 		.driver = {
-			.name = "i.MX-sc-tsens",
+			.name = "imx-sc-thermal",
 			.of_match_table = imx_sc_tsens_table,
 		},
 };
-- 
1.7.9.5

